### PR TITLE
use const where possible and remove vars

### PR DESCRIPTION
**Pull request by craigmichaelmartin**
:)

### CODE REVIEW

File: js/stores/windowStore.js [hunk 1]
```diff
@@ -313,8 +313,8 @@ const doAction = (action) => {
       windowStore.emitChange()
       break
     case WindowConstants.WINDOW_TAB_MOVE:
-      let sourceFramePropsIndex = FrameStateUtil.getFramePropsIndex(windowState.get('frames'), action.sourceFrameProps)
-      let newIndex = FrameStateUtil.getFramePropsIndex(windowState.get('frames'), action.destinationFrameProps) + (action.prepend ? 0 : 1)
+      const sourceFramePropsIndex = FrameStateUtil.getFramePropsIndex(windowState.get('frames'), action.sourceFrameProps)
+      const newIndex = FrameStateUtil.getFramePropsIndex(windowState.get('frames'), action.destinationFrameProps) + (action.prepend ? 0 : 1)
```
Comment (bbondy) [inline js/stores/windowStore.js]: I think this one still needs to be `let`, see below with `newIndex--`

**Comment by bbondy** [comment]
Thanks so much for the cleanup! Please adjust the one instance I found and also rebase for easier merging. Thanks again!

File: js/stores/windowStore.js (hunk 1 above)
Comment (craigmichaelmartin) [inline js/stores/windowStore.js]: Yeah, `standard` caught it, I fixed it, and neglected to add it :disappointed:

**Comment by craigmichaelmartin** [comment]
Sure thing!

**Comment by bbondy** [comment]
Thanks for the cleanup, merging.

**Pull request merged**